src/main/java/fashionable/simba/yanawaserver/matching/domain/Level.java

```diff
@@ -9,7 +9,7 @@ public class Level {
     private final double level;
 
     private static final HashSet<Double> levels = new HashSet<>(
-        Arrays.asList(0.0, 0.5, 1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5, 6.0));
+            Arrays.asList(0.0, 0.5, 1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5, 6.0));
```

Comment:
블럭 분리 단위를 잘 맞춰야 가독성이 높아져요! ☺️
```suggestion
            Arrays.asList(0.0, 0.5, 1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5, 6.0)
          );
```